dace/codegen/targets/cpu.py

```diff
@@ -588,6 +588,9 @@ def _emit_copy(
         elif isinstance(src_node, nodes.CodeNode) and isinstance(dst_node, nodes.CodeNode):
             # Code->Code copy (not read nor write)
             raise RuntimeError("Copying between code nodes is only supported as part of the participating nodes")
+        elif uconn == None and vconn == None and memlet.data == None and dst_schedule is dtypes.ScheduleType.Sequential:
```

Comment:
all of the tests here are inverted. `is None` and `== <enum entry>` is what you should use.

Reply:
Thanks! Fixed it